pkgs/development/python-modules/pytorch-tokenizers/default.nix

```
@@ -90,5 +90,9 @@ buildPythonPackage rec {
     homepage = "https://github.com/meta-pytorch/tokenizers";
     license = lib.licenses.bsd3;
     maintainers = with lib.maintainers; [ GaetanLepage ];
+    badPlatforms = [
+      # See https://github.com/NixOS/nixpkgs/issues/466092
```

```suggestion
      # TODO
      # See https://github.com/NixOS/nixpkgs/issues/466092
```
Please, add a comment explaining the reason of the build failure.

---

Done. I'll go through and do the rest.